bio_hansel/const.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+FASTA_COLUMNS_TO_REMOVE = '''
+pident
+length
+mismatch
+gapopen
+qstart
+qend
+etc
```

Comment:
Hi Matt,
Please remove the following columns from the detailed report:
`pident  length  mismatch    gapopen qstart  qend    sstart  send    evalue  bitscore    qlen    slen    seq coverage    is_trunc`

Thanks

Reply:
Thanks for the clarification, I'll make those changes.